blacklight-cornell/config/initializers/assets.rb

```diff
@@ -10,4 +10,5 @@
 Rails.application.config.assets.paths << Rails.root.join('vendor', 'assets', 'images')
 
 Rails.application.config.assets.precompile += %w( aeon.css )
-Rails.application.config.assets.precompile += %w( aeon.js )
\ No newline at end of file
+Rails.application.config.assets.precompile += %w( aeon.js )
+Rails.application.config.assets.precompile += %w( search_form.js )
```

Comment:
Could probably combine these precompiled assets into one list if you wanted, i.e. `%w( aeon.css aeon.js search_form.js )`